idunn/api/places_list.py

```diff
@@ -123,26 +132,38 @@ def valid_categories(cls, v):
         return v
 
 
-def get_raw_params(query_params):
-    raw_params = dict(query_params)
-    if 'category' in query_params:
-        raw_params['category'] = query_params.get_list('category')
-    if 'raw_filter' in query_params:
-        raw_params['raw_filter'] = query_params.get_list('raw_filter')
+def get_raw_params(bbox, category, raw_filter, source, q):
+    raw_params = {
+        'bbox': bbox,
+        'category': category,
+        'raw_filter': raw_filter,
+        'source': source,
+        'q': q,
+    }
     if raw_params.get('raw_filter') and raw_params.get('category'):
-        raise BadRequest(
-            detail={"message": "Both \'raw_filter\' and \'category\' parameters cannot be provided together"}
+        raise HTTPException(
+            status_code=400,
+            detail="Both \'raw_filter\' and \'category\' parameters cannot be provided together"
         )
     return raw_params
 
-def get_places_bbox(bbox, es: Elasticsearch, indices: IndexNames, settings: Settings, query_params: http.QueryParams):
-    raw_params = get_raw_params(query_params)
+
+def get_places_bbox(
+    bbox: Any,
+    category: Optional[str] = Query(None),
+    raw_filter: Optional[str] = Query(None),
```

Comment:
Is this correct, given that this route should accept multiple `category` or `raw_filter` in the same query ?